libgstc/libgstc.c

```diff
@@ -365,6 +365,51 @@ gstc_client_ping (GstClient * client)
   return gstc_cmd_send (client, request);
 }
 
+GstcStatus
+gstc_element_get (GstClient * client, const char *pname,
+    const char *element, const char *parameter, const char *format, ...)
+{
+  GstcStatus ret;
+  va_list ap;
+  const char *what_fmt = "/pipelines/%s/elements/%s/properties/%s";
+  char *what;
+  char *response;
+  char *out;
+
+  gstc_assert_and_ret_val (client != NULL, GSTC_NULL_ARGUMENT);
+  gstc_assert_and_ret_val (pname != NULL, GSTC_NULL_ARGUMENT);
+  gstc_assert_and_ret_val (element != NULL, GSTC_NULL_ARGUMENT);
+  gstc_assert_and_ret_val (parameter != NULL, GSTC_NULL_ARGUMENT);
+  gstc_assert_and_ret_val (format != NULL, GSTC_NULL_ARGUMENT);
+
+  va_start (ap, format);
+
+  asprintf (&what, what_fmt, pname, element, parameter);
+
+  ret = gstc_cmd_read (client, what, &response);
+  if (ret != GSTC_OK) {
+    goto unref;
+  }
+
```

Comment:
Collapse this multiple empty lines.

Reply:
This has been done in the latest update (lines below this)